storage/azure/src/frontend/FrontendInterfaces.ts

```diff
@@ -6,17 +6,11 @@ import {
   FrontendConfigDownloadInput,
   FrontendConfigUploadInput,
   FrontendUploadInMultiplePartsInput,
-  TransferConfig,
 } from "@itwin/object-storage-core/lib/frontend";
 
-export interface AzureTransferConfig extends TransferConfig {
-  authentication: string;
-}
-
-export interface AzureTransferConfigInput {
-  transferConfig: AzureTransferConfig;
-}
+import { AzureTransferConfigInput } from "../common";
 
+/* eslint-disable @typescript-eslint/indent */
```

Comment:
most likely this comes from `@itwin/eslint-recommended` - we don't use Prettier so we rely on ESLint both for code quality and formatting. @gdatbentley can you please switch the configs back so that Prettier overrides ours?

Reply:
Yeah looks like I shot myself in the foot by changing that, the original eslintrc seems to work out better now